index.js

```diff
@@ -0,0 +1,115 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex')
+const cors = require('cors')
+const server = express();
+const bcrypt = require('bcryptjs');
+const dbConfig = require('./knexfile')
+const jwt = require('jsonwebtoken')
+const session = require('express-session');
+const jwtKey = require('./secret/keys').jwtKey;
+
```

Comment:
Both cookies/sessions & JWT's try to solve the same problem (_having the server remember/recognize a client over HTTP, a stateless (CRS) protocol_). This means _you only need to use one or the other_. 

I think JWT's are easier to deal with in my opinion, and your code is more or less already set up for that, so I suggest removing session-based code.